corehq/apps/hqmedia/tasks.py

```diff
@@ -126,10 +126,18 @@ def build_application_zip(include_multimedia_files, include_index_files, domain,
                           download_targeted_version=False):
     DownloadBase.set_progress(build_application_zip, 0, 100)
     app = get_app(domain, app_id)
-    fpath = create_files_for_ccz(app, build_profile_id, include_multimedia_files, include_index_files,
-                                 download_id, compress_zip, filename, download_targeted_version,
-                                 task=build_application_zip, expose_link=True)
-    _expose_download_link(fpath, filename, compress_zip, download_id)
```

Comment:
Non-blocking: I'd have kept this line and removed the `expose_link` param, which looks like it's never passed as True except for here, for the sake of having one fewer param.

Reply:
`create_files_for_ccz` is actually only used in two places, and both of them set it to `True`, so I just dropped the param.